docs/tctl.md

```diff
@@ -269,9 +269,10 @@ You can reset to some predefined event types:
 ./tctl workflow reset -w <wid> -r <rid> --reset_type <reset_type> --reason "some_reason"
 ```
 
-- FirstDecisionCompleted: reset to the beginning of the history.
-- LastDecisionCompleted: reset to the end of the history.
+- FirstWorkflowTask: reset to the beginning of the history.
+- LastWorkflowTask: reset to the end of the history.
 - LastContinuedAsNew: reset to the end of the history for the previous run.
+- BadBinary: <>.
 
 If you are familiar with the Temporal history event, You can also reset to any decision finish event by using:
```

Comment:
"decision" is used here too. Let's CTRL-F the entire file for decision and replace where necessary